transports/tcp/CHANGELOG.md

```diff
@@ -17,9 +17,12 @@
 
 - Update `rust-version` to reflect the actual MSRV: 1.60.0. See [PR 3090].
 
+- Optimize listeners polling behaviour in the tcp transport, fixing [2781](https://github.com/libp2p/rust-libp2p/issues/2781). See [PR 3361].
```

Comment:
Do we need this? It is mostly a refactoring that is not user facing.

Could mention the bug fix but otherwise I'd suggest not adding a changelog entry.

Reply:
removed.